_guides/project-boards/configure.md

```diff
@@ -285,11 +287,81 @@ The project board will automatically refresh.  The cards in the completed column
 <p>You may have noticed that we didn't provide an alternate template for cards that aren't completed.  If our custom template is blank, Cerb uses its own default template.</p>
 </div>
 
+### Using conditional logic with tasks
+
+Not every task is completed on time; what if we want to add a visual cue when a task is  past due?
+
+1. On that same profile go ahead and click on the gear button (⚙️ Edit) at the top of the profile page.
+1. Scroll down to **Task** and click into the **Card custom template** text box.
+1. Paste the following template before line 7:
+	<pre>
+	<code class="language-javascript line-numbers">
+	{% raw %}    
+    {% else %}
+    
+    {% if date(due) < date() %}
+    
+    &lt;div style="font-size:120%;"&gt;
+		&lt;span class="glyphicons glyphicons-clock" style="color:rgb(150,0,0);"&gt;&lt;/span&gt; Overdue
+	&lt;/div&gt;
+    
+    {% endif %}
+	{% endraw %}
+	</code>
+	</pre>
+
+{% comment%}<div class="cerb-box note">
+<p>If you are modifying the same task from earlier, you will need to either completely replace what was pasted into the template editor or insert the "elseif" block between the original if-statement and the endif-statement.</p>
+</div>{% endcomment%}
+
+Now we have two different visual cues. Overdue will only be active if the task is not already complete **and** the task was already due. We changed the icon color to red to make the cue seem less pleasant and changed the icon to a clock due to the nature of the cue.
+
+Let's break down how to manually modify each of the sections of code we just blindly copy and pasted into the template editor from earlier:
+
+1. The beginning of each visual cue has some form of an if-statement because we don't want every task to be modified with our visual cue. The if-statement executes if the boolean within it calculates to <i>true</i>. This boolean statement can be on anything (but we suggest you make it related to the task at hand). 
```

Comment:
* Switch `if-statement` to `"if" statement".
* You repeat the phrase "visual cue" twice in the same opening sentence.
* `true` is using HTML formatting rather than Markdown (it should be back-ticks for code, rather than italicized).
* The last sentence doesn't add any useful information.